src/ui/components/NetworkMonitor/utils.ts

```diff
@@ -149,20 +167,14 @@ export const partialRequestsToCompleteSummaries = (
         return true;
       }
 
-      if (types.has(row.type)) {
-        return true;
-      }
-      if (types.has("xhr") && row.type.match(/json/)) {
-        return true;
+      if (!row.cause) {
+        return false;
```

Comment:
Instead of this, I think we should do

```
if (types.has(REQUEST_TYPES[row.cause || "other"])) {
```

Reply:
Actually better yet
```
if (types.has(REQUEST_TYPES[row.cause] || REQUEST_TYPES.other)) {

Reply:
Oh, this wasn't to handle `Other`, that should be handled already, this is the case where for some reason the cause is `undefined`, but I'm realizing now that is impossible and I should just put that in the type system.

Reply:
`request.event.requestCause` can be `undefined` so that's the main thing to keep in mind. The ternary should cover it though.

Reply:
now this can be removed seems like.